File: python/leetcode027.py
class Solution:
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """

        l = ls = len(nums)
        for i in range(ls-1, -1, -1):
            if nums[i] == val:
                nums.pop(i)
                l = l-1
        return l

File: leetcodec++/leetcode027.cpp
//
// Created by 黄敏辉 on 2019-05-06.
//

class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int i = 0;
        while (i < nums.size()){
            if (nums[i] == val){
                nums.erase(nums.begin() + i);
            }else
                i++;
        }
        return nums.size();
    }
};